src/api/__tests__/Feed.test.js

```diff
@@ -1369,6 +1369,21 @@ describe('api/Feed', () => {
             expect(feed.updateFeedItem).toHaveBeenCalled();
             expect(successCallback).toHaveBeenCalled();
         });
+
+        test('should call updateFeedItem and feedErrorCallback on failure', () => {
+            const e = new Error('foo');
+            feed.updateFeedItem = jest.fn();
+            feed.feedErrorCallback = jest.fn();
+            feed.updateAnnotation = jest.fn().mockImplementation(() => {
+                feed.updateFeedItem();
+                feed.feedErrorCallback(true, e, errorCode);
+            });
+
+            feed.updateAnnotation(file, annotationId, permissions, text, successCallback, errorCallback);
+
+            expect(feed.updateFeedItem).toHaveBeenCalled();
+            expect(feed.feedErrorCallback).toHaveBeenCalledWith(true, e, errorCode);
```

Comment:
This is testing mocked `feed.updateAnnotation` not the real code. Since you're testing `feed.updateAnnotation`, you shouldn't mock it, right? How about try to mock `feed.annotationsAPI.updateAnnotation`?

Reply:
I don't think I need this test anymore because I've moved out the callback to a helper method. I think since I am using the helper in two locations, it makes sense to have a helper rather than hardcode the logic in both locations.